site/profile/manifests/puppet/pe_agent.pp

```diff
@@ -1,8 +1,13 @@
 # This class manages tp::test for PE Agents
 #
 class profile::puppet::pe_agent (
-  Boolean $test_enable = false,
-  Hash $settings       = {},
+  Boolean $test_enable        = false,
```

Comment:
Please make relevant MR to puppet-psick, here as in the other classes in old profile